src/globus_sdk/client.py

```diff
@@ -111,7 +111,7 @@ def get(
         self,
         path: str,
         *,
-        params: Optional[Dict] = None,
+        query_params: Optional[Dict[str, str]] = None,
```

Comment:
I'll just comment on this first case, but this holds through.
I think `Dict[str, Any]` is better for these.

That handles integers and bools gracefully. True, it allows for incorrect things as well, but mypy is always giving us trade-offs.

Based on what's done in the stdlib and typeshed, I think this is a good case for Any.